pkg/backup/compaction_job.go

```diff
@@ -492,22 +492,12 @@ func updateCompactionBackupDetails(
 	compactionChain compactionChain,
 	initialDetails jobspb.BackupDetails,
 	resolvedDest backupdest.ResolvedDestination,
-	encryption *jobspb.BackupEncryptionOptions,
+	baseEncryptOpts *jobspb.BackupEncryptionOptions,
 	kmsEnv cloud.KMSEnv,
 ) (jobspb.BackupDetails, error) {
 	if len(compactionChain.chainToCompact) == 0 {
 		return jobspb.BackupDetails{}, errors.New("no backup manifests to compact")
 	}
-	var encryptionInfo *jobspb.EncryptionInfo
```

Comment:
I think we can also update `MakeNewEncryptionOptions` to use `encryption.Unencrypted()` too.

Reply:
Nice idea. done.